MagicExt-OAuth.podspec

```diff
@@ -21,7 +21,7 @@ TODO: Add long description of the pod here.
 
   s.source_files = 'Sources/MagicExt-OAuth/**/*'
 
-  s.dependency 'MagicSDK', '~> 9.0'
+  s.dependency 'MagicSDK', '~> 10.0'
```

Comment:
We should include this change for OIDC, so that it may be used with the latest version of Magic SDK.